docs/explanation/generic-rules.md

```diff
@@ -0,0 +1,68 @@
+end
+
+grafana-agent ---|prometheus_remote_write| prometheus
+cos-proxy ---|metrics_endpoint| prometheus
+```
+You can find more information on these groups and the alert rules they contain below.
+
+## `HostHealth` alert group
+The `HostHealth` alert rule group contains the `HostDown` and `HostMetricsMissing` alert rules, identifying unreachable target scenarios.
+
+### `HostDown` alert rule
+The purpose of this alert is to notify when Prometheus (or Mimir) failed to scrape its target. The alert expression executes `up{...} < 1` with labels including the target's Juju topology: `juju_model`, `juju_application`, etc. In essence, this alert notifies you in case of failed scrapes and is directly applicable to cases where a charm is being scraped by Prometheus for metrics. The [`up` metric](https://prometheus.io/docs/concepts/jobs_instances/), which is what this alert's expression relies on, indicates the health or reachability status of a node. For example, when `up` is 1 for a charm, this is a sign that Prometheus is able to successfully call the metrics endpoint of that charm and access the metrics that are exposed at that endpoint.
+
+This alert is especially important for Cos Lite, where Prometheus is capable of scraping charms for metrics. The firing of this alert indicates that Prometheus is not able to scrape a target for metrics, leading to `up` being 0.
+
+
+### `HostMetricsMissing`
+```{note}
+`HostMetricsMissing` is also used in the `AggregatorHostHealth` group. As part of the `HostHealth` group, however, it monitors the health of all charms whose metrics are collected by an aggregator and then remote written to a metrics backend and not just aggregators (see the `AggregatorHostHealth` group for details on the distinction).
+```
+
+The purpose of this alert is to notify when metrics are not reaching the Prometheus (or Mimir) database, regardless of whether scrape succeeded. The alert expression executes `absent(up{...})` with labels including the aggregator's Juju topology: `juju_model`, `juju_application`, `juju_unit`, etc. 
+
+Like the `HostDown` rule, the purpose of this rule is to give you an idea of the health of a node and whether it is reachable. However, unlike `HostDown`, `HostMetricsMissing` is used in scenarios where metrics from a charm are remote written into Prometheus or Mimir, as opposed to being scraped. This rule is especially important in COS, where the use of Mimir instead of Prometheus warrants metrics to be remote written (as Mimir does not scrape).
+
+To provide an example to distinguish between `HostDown` and `HostMetricsMissing`, consider how the health of Alertmanager can be monitored in different deployment scenarios below:
+
+- In Cos Lite, metrics from Alertmanager are scraped by Prometheus. A successful scrape results in `up` = 1 and a failed scrape results in `up` = 0. As a result, we can rely on the `HostDown` alert to notify in case of issues with the scrape i.e. when `up` is 0.
```

Comment:
```suggestion
- In COS Lite, metrics from Alertmanager are scraped by Prometheus. A successful scrape results in `up` = 1 and a failed scrape results in `up` = 0. As a result, we can rely on the `HostDown` alert to notify in case of issues with the scrape, i.e. when `up` = 0.
```
- Changed to = in the last one to stay consistent with the previous notation